src/NServiceBus.Core.Tests/Faults/MoveFaultsToErrorQueueTests.cs

```diff
@@ -39,7 +38,7 @@ public void Setup()
                 await behavior.Invoke(context, () => Task.FromResult(0), c => CaptureFaultContext(c, out faultContext));
             }
 
-            Assert.AreEqual("errors-queue", faultContext.ErrorQueueAddress);
+            Assert.AreEqual("some-source-queue", faultContext.SourceQueueAddress);
```

Comment:
Yes, that makes sense. It's based through the behaviour to the context, so it should be tested. Fixed